src/domains/profile/components/ResetProfile/ResetProfile.tsx

```diff
@@ -22,7 +22,7 @@ export const ResetProfile = ({ isOpen, profile, onClose, onCancel, onReset }: Re
 	const { persist } = useEnvironmentContext();
 
 	const handleReset = async () => {
-		profile.initializeSettings();
+		profile.flush();
```

Comment:
This method will also delete all data. Is that intended here?

Reply:
@faustbrian we need to reset to defaults. Noticed that `ProfileInitializer` is re-initializing settings and erasing modified profile data, same as with `initializeSettings` we used previously. Can you confirm?

Reply:
`initializeSettings` previously only reset settings but not the data. I'll make a release tomorrow that restores the old behaviour of having 2 separate methods.